### PR TITLE
Update event list

### PR DESCRIPTION
**Pull request by gnunicorn**

### CODE REVIEW

**Comment by gnunicorn** [comment]
@kaichaosun , @JoshOrndorff please confirm I calculated the remote times correctly. I attempted to go for localised time or UTC, however we are also switching from CEST to CST end of october, so I am not sure the times you had put in did actually match up. Please confirm these time slots work for you now.

File: README.md [hunk 1]
```diff
@@ -24,21 +24,21 @@ Join us for Hacktoberfest 2019. Hack on parity and related technologies, solve i
...
+- **Oct 14th** [22:00 CST](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Substrate+Collaborative+Learning+Hacktoberfest+edition&iso=20191014T22&p1=237&ah=2) · Substrate Collaborative Learning Hacktoberfest edition (2 hours, https://zoom.us/j/440029011)
+- **Oct 15th** [18:30 CEST](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Substrate'n'Chill+Hacktoberfest-Edition&iso=20191015T1830&p1=37) · Substrate'n'Chill Hacktoberfest-Edition · Berlin Community Space
+- **Oct 18th** [22:00 CST](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Substrate+Collaborative+Learning+Hacktoberfest+edition&iso=20191018T22&p1=237&ah=2) Substrate Collaborative Learning Hacktoberfest edition (2 hours, [via Matrix](https://matrix.to/#/!trdlqNGrCsZpYUZoXa:matrix.parity.io?via=matrix.parity.io&via=matrix.org)) with Ben & Kaichao
```
Comment (kaichaosun) [inline README.md]: This one should be 12:00 UTC

Comment (gnunicorn) [reply README.md]: you mean `20:00 CST` or I should write `12:00 UTC`? I picked CST for the ones we picked to target the asien audience ...

Comment (gnunicorn) [reply README.md]: updated.

**Pull request merged**